### PR TITLE
PR_hometask3

### PR DESCRIPTION
**Pull request by vvasilkova**
Hello!

### CODE REVIEW

File: tasks/condition/condition.js [hunk 1]
```diff
@@ -5,34 +5,80 @@
...
+  const x = 3;
+  const y = 7;
+}
+// Task 1 - if else:
+{
+  const x = 3;
+  const y = 7;
+
+  let z;
+  if ((x < 10) && (y < 10)) {
```
Comment (dshaplyko) [inline tasks/condition/condition.js]: inner brackets are not necessary, 
if ( x < 10  &&  y < 10 )

File: tasks/condition/condition.js [hunk 2]
```diff
@@ -5,34 +5,80 @@
...
+    console.log('No results');
+  }
 }
+// Task 2 - ternary operator:
+{
+  const x = 3;
+  const y = 7;
 
+  let z;
+  ((x < 10) && (y < 10)) ? console.log('z =', x + y):
```
Comment (dshaplyko) [inline tasks/condition/condition.js]: too superfluous
let z = ( x < 10  &&  y < 10 ) ? x + y: x * y;
console.log(z);

File: tasks/cycles/cycles.js [hunk 3]
```diff
@@ -42,6 +113,13 @@ function calculateDiscount(redemption) {
     interestOnDeposit: 250,
     otherExpences: -300
   };
+  var totalIncome = 0;
+  var x;
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js]: remove this

File: tasks/cycles/cycles.js [hunk 4]
```diff
@@ -42,6 +113,13 @@ function calculateDiscount(redemption) {
...
+  for (x in personIncomes) {
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js]: define the variable inside the loop
for ( let x in personIncomes )

File: tasks/cycles/cycles.js [hunk 5]
```diff
@@ -14,24 +14,95 @@ function calculateDiscount(redemption) {
...
+    discount = 0;
+  } else if (redemption >= 351 && redemption <= 1350) {
+    discount = 15;
+  } else if (redemption >= 1351 && redemption <= 2700) {
+    discount = 30;
+  } else if (redemption >= 2701 && redemption <= 6500) {
+    discount = 45;
+  } else if (redemption >= 6501 && redemption <= 9999) {
+    discount = 60;
+  }
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js:27]: you do not consider a case when the redemption is a string.

**Review by dshaplyko — CHANGES_REQUESTED** [review]
Please consider my comments. Also, please see my email explaining your error

File: tasks/condition/condition.js (hunk 1 above)
Comment (vvasilkova) [reply tasks/condition/condition.js]: Done

File: tasks/condition/condition.js (hunk 2 above)
Comment (vvasilkova) [reply tasks/condition/condition.js]: Done

File: tasks/cycles/cycles.js (hunk 3 above)
Comment (vvasilkova) [reply tasks/cycles/cycles.js]: var x removed

File: tasks/cycles/cycles.js (hunk 4 above)
Comment (vvasilkova) [reply tasks/cycles/cycles.js]: Done

File: tasks/cycles/cycles.js (hunk 5 above)
Comment (vvasilkova) [reply tasks/cycles/cycles.js:27]: Done

**Pull request merged**